testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+    public void fail_if_iterable_size_is_not_equal() {
+        Verify.assertSize(-1, FluentIterable.of(1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_null_in_assert_size() {
+        Verify.assertSize(0, (Iterable) null);
+    }
+
+    @Test
+    public void pass_if_iterable_size_is_equal() {
+        Verify.assertSize(0, FluentIterable.of());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_size_is_not_equal() {
+        Verify.assertSize(-1, Collections.emptyMap());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_assert_size() {
+        Verify.assertSize(0, (Map) null);
+    }
+
+    @Test
+    public void pass_if_map_size_is_equal() {
+        Verify.assertSize(0, Collections.emptyMap());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_size_is_not_equal() {
+        Verify.assertSize(-1, ArrayListMultimap.create());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_null_in_assert_size() {
+        Verify.assertSize(0, (Multimap) null);
+    }
+
+    @Test
+    public void pass_if_multimap_size_is_equal() {
+        Verify.assertSize(0, ArrayListMultimap.create());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_collection_size_is_not_equal() {
+        Verify.assertSize(-1, Collections.emptyList());
+    }
+
+    @Test
+    public void pass_if_collection_size_is_equal() {
+        Verify.assertSize(0, Collections.emptyList());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_string_not_contains_char_sequence() {
+        Verify.assertContains(notEmptyString, emptyString);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_char_sequence_is_null_in_contains() {
+        Verify.assertContains(null, emptyString);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_string_is_null_in_contains() {
+        Verify.assertContains(emptyString, (String) null);
+    }
+
+    @SuppressWarnings({"ConstantConditions", "ErrorNotRethrown"})
+    @Test(expected = AssertionError.class)
+    public void fail_if_contains_char_sequence_or_string_is_null() {
+        final String nullString = null;
+
+        try {
+            Verify.assertContains(null, emptyString);
+        } catch (AssertionError e) {
+            Verify.assertContains(emptyString, nullString);
+        }
+    }
+
+    @Test
+    public void pass_if_string_contains_char_sequence() {
+        Verify.assertContains(emptyString, notEmptyString);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_is_string_contains_char_sequence() {
```

Comment:
Something wrong with the method name. It neither makes sense nor reflects the test case.